src/gpu/intel/jit/reorder/gen_reorder.cpp

```diff
@@ -158,7 +159,8 @@ status_t gen_reorder_t::pd_t::init(impl::engine_t *engine,
     cfg->set_zp_cfg(zp_cfg);
     VDISPATCH_REORDER_SC(
             init_kernel_info(), "kernel initialization unsuccessful");
-
+    VDISPATCH_REORDER_SC(
+            maybe_create_zp_precompute_conv_pd(dst_engine), VERBOSE_BAD_FLAGS);
```

Comment:
I think "bad flags" here is confusing - a lot of things can go wrong when creating convolution. I would use a custom message, like "failed to create nested convolution for precomputed zero-points"

There is one more similar message below.